test/intl402/DateTimeFormat/prototype/resolvedOptions/offset-timezone-change.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-createdatetimeformat
+description: test offset timezone with sign sync with resolvedOptions().timeZone.
```

Comment:
```suggestion
description: Tests that offset time zones are correctly normalized in resolvedOptions() output.
```
And it might just make sense to merge this file and test/intl402/DateTimeFormat/prototype/resolvedOptions/offset-timezone-basic.js .